package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "async": "2.0.0-rc.5",
```

Comment:
remarkable depends on it, but does not install it by itself. so we need this here